kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
I wonder if this could be worked around by something like internally in Hawkular/Accounts always using a combo of some virtual host/port, which then is mapped by Docker itself via properties or such to the used ones. 

The issue is that e.g. for the CD we will never know in advance what ip/port we will get. Or when users just `docker run` hawkular. So we need the help of Docker to install the correct values.